sql/sql_class.cc

```diff
@@ -718,8 +718,7 @@ THD::THD(bool enable_plugins)
       m_is_plugin_fake_ddl(false),
       m_inside_system_variable_global_update(false),
       bind_parameter_values(nullptr),
-      bind_parameter_values_count(0),
-      is_rpl_stmt_event_format_used(true) {
+      bind_parameter_values_count(0) {
```

Comment:
Same as above

Reply:
Yes, I fixed all occurrences of is_rpl_stmt_event_format_used.